src/webob/cookies.py

```diff
@@ -239,10 +240,13 @@ def serialize_cookie_date(v):
         v = timedelta(seconds=v)
 
     if isinstance(v, timedelta):
-        v = datetime.utcnow() + v
+        v = datetime.now(tz=zoneinfo.ZoneInfo("UTC")) + v
```

Comment:
The string formatting below doesn’t care that the datetime object is aware here.